modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -71,9 +85,9 @@ It is possible to create conflicting quotas if you configure the same quotas thr
 
 . Quota configured through the Kafka API for an exact match on `client_id`
 . Quota configured through the Kafka API for a prefix match on `client_id`
-. Quota configured through cluster configuration properties (`kafka_client_group_byte_rate_quota`, `kafka_client_group_fetch_byte_rate_quota`, xref:upgrade:deprecated/index.adoc[deprecated starting in v24.2]) for a prefix match on `client_id`
+. Quota configured through cluster configuration properties (`kafka_client_group_byte_rate_quota`, `kafka_client_group_fetch_byte_rate_quota`-deprecated in v24.2) for a prefix match on `client_id`
```

Comment:
@micheleRP this should not be in cloud. Anything related to cluster configs just skip.  

This includes both this stuff and  the whole 'migration' section below. There's nothing to migrate from in cloud since they never had access to the old cluster config based quotas